package.json

```diff
@@ -2,5 +2,8 @@
   "name": "cellxgene",
   "scripts": {
     "postinstall": "npm install --prefix client && npm run build --prefix client && make copy-client-assets"
+  },
```

Comment:
why is there a package.json at both top-level and in client/ directory?

Reply:
Removed.

Reply:
oh, weird - there is also a package.json at the top level.   And they are also in main (you didn't add them).

This came from a PR @MDunitz added a few months ago.  #1427     @MDunitz - any idea why we need two package.jsons?

Reply:
It's all good — this may have been a misfire — I have done that. If you run npm start in the wrong folder, it will blow up. If you run `npm install foo`, it will start implicitly creating things and add a lock file. That can cause some confusion and may have been what happened?